Ryujinx.Ui.Common/Configuration/ConfigurationState.cs

```diff
@@ -1096,6 +1098,22 @@ public void Load(ConfigurationFileFormat configurationFileFormat, string configu
                 configurationFileUpdated = true;
             }
 
+            if (configurationFileFormat.Version < 39)
+            {
+                Ryujinx.Common.Logging.Logger.Warning?.Print(LogClass.Application, $"Outdated configuration version {configurationFileFormat.Version}, migrating to version 39.");
+
+                configurationFileFormat.Hotkeys = new KeyboardHotkeys
+                {
+                    ToggleVsync = configurationFileFormat.Hotkeys.ToggleVsync,
+                    Screenshot = configurationFileFormat.Hotkeys.Screenshot,
+                    ShowUi = configurationFileFormat.Hotkeys.ShowUi,
+                    Pause = configurationFileFormat.Hotkeys.Pause,
+                    ToggleMute = configurationFileFormat.Hotkeys.ToggleMute,
+                    ResScaleUp = configurationFileFormat.Hotkeys.ResScaleUp,
+                    ResScaleDown = configurationFileFormat.Hotkeys.ResScaleDown
```

Comment:
I think this is wrong and should be this
```suggestion
                    ResScaleUp = Key.Unbound,
                    ResScaleDown = Key.Unbound
```

Reply:
Right you are!